index.html

```diff
@@ -46,6 +46,207 @@ <h1>I'm Naan.
+                        <div class ="primary_text">
+                            <div class="project_title "> Multi-Post Stories</div>
+                            <div class="sub_header">
+                                <div class="client">
+                                    <div class="client_text ">Canopy</div>
+                                </div>
+                                <div class="counter"></div>
+                                <div class="role">
+                                    <div class="text underline">Back End Dev</div>
+                                </div>
+                                <div class="counter"></div>
+                                <div class="year_label">
+                                    <div class="text underline">2015</div>
+                                </div>
+                            </div>
+                        </div>
+                        <div class ="primary_text_1">A daily selection of 
+                            privately personalized reads; no accounts or 
+                            sign-ups required.
+                        </div>
+                        <ul class="tags">
+                            <li class="tag_1 tag_html">html</li>
+                            <li class="tag_2 tag_css">css</li>
+                            <li class="tag_3 tag_javascript">javascript</li>
+                        </ul>
+                        <div class="actions">
+                            <div class="buttons">
+                                <button class="see_project">See project</button>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+            <div>
+                <div class="about_myself">
+                    <div class="top underline">
+                        <div class="about_header underline">About Myself</div>
+                        <div class="about_description ">
+                            Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! 
+                            If you like what you see and have a project you need coded, don’t hestiate to contact me.
+                        </div>
+                        <div class="social_media">
+                            <div class="supporting_text underline">
+                                <div class="connect ">LET’S CONNECT</div>
+                            </div>
+                        </div>
+                        <div class="icon_container border">
+                            <div><img src="images/github.jpg"></div>
+                            <div><img src="images/linkedin.jpg"></div>
+                            <div><img src="images/reddit.jpg"> </div>
+                            <div><img src="images/twitter.jpg"> </div>
+                            <div><img src="images/medium.jpg"> </div>
```

Comment:
- **[OPTIONAL]** Always when you list some objects the correct way is to use the `ul` tag for better practices of HTML & CSS, please 🙏🏽 use the `ul` tag to list the social media icons.